lambda/rotator-snowflake/setup/okta/okta_test.go

```diff
@@ -19,8 +19,6 @@ func TestGetOktaClient(t *testing.T) {
 	os.Setenv("OKTA_PRIVATE_KEY", "testPrivKey")
 	os.Setenv("OKTA_ORG_URL", "https://www.testOrgURL.com")
 	os.Setenv("OKTA_CLIENT_ID", "testClientID")
-	os.Setenv("OKTA_DATABRICKS_APP_ID", "databricksAppID")
-	os.Setenv("OKTA_SNOWFLAKE_APP_IDS", "snowflakeAppID1,snowflakeAppID2,snowflakeAppID3")
```

Comment:
okta app IDs are now specific to either databricks or snowflake, not prefixed with "okta"